runtime/tr.source/trj9/compile/J9Compilation.cpp

```diff
@@ -83,11 +83,11 @@ void *operator new(size_t size)
 /**
  * Since we are using arena allocation, heap deletions must be a no-op, and
  * can't be used by JIT code, so we inject an assertion here.
- */
+ *
 void operator delete(void *)
    {
    TR_ASSERT(0, "Invalid use of global operator delete");
-   }
+   }*/
```

Comment:
Was there a reason this was commented out?

Reply:
The Java 9 libraries were tripping debug builds because of operator new and delete in non JIT libraries from the JCL - removed from the commits.

Reply:
Do you mind opening an issue so we can take a look at it at some point :).

Reply:
Opened #374 to track this one - not sure of the 'correct' fix, but put some ideas in the issue.